src/components/CvssCalculator/CvssVectorInput.vue

```diff
@@ -62,11 +62,11 @@ const getFactorColor = (weight: number, isHovered: boolean = false) => {
       class="osim-cvss-score"
       :style="isFocused ? {color: 'white'} : {color: 'black'}"
     >
-      {{ cvssScore != null ? `${cvssScore} ` : '' }}
+      {{ cvssScore !== null ? `${cvssScore} ` : '' }}
```

Comment:
I wonder if this strict null check could be problematic...
I tried to follow the usage and seems to be handled, the only non null expression (I found) is this one:
https://github.com/RedHatProductSecurity/osim/blob/b24b26dd6b943ae346381fada87d6b9b3f2feff8/src/composables/useCvssScores.ts#L152
In the case the `rhCvss` is not a ref ordoes not have `score`, is going to return `undefined`, hopefully this should not happen, as the shape ofthe `rhCvss` is hardcoded, but we should keep it in mind.

We could use the nullish coalescing there as well
```js
const cvssScore = computed(() => rhCvss.value?.score ?? null); 
```